### DISCUSSION TITLE
Github actions group for effective CI/CD

### DISCUSSION THREAD

**Discussion by manickavela29** [k2-fsa/sherpa-onnx#971] [Q&A] [answered]
Observed that github actions configure files (*.yaml under .github/workflows/configfiles.yaml)
Each file is having unique concurrency parameter.
which is not at all effective

https://docs.github.com/en/actions/using-jobs/using-concurrency

sample : 
` concurrency: 
group: build-wheels-win32-${{ github.ref }}
cancel-in-progress: true`

It would be great if it is ordered by groups of platform.
stype_test can also have unique group by so it will come first and 
developers need not wait for a long time for this

Its a very minor suggestion and work, 
It can be under 'help-wanted' maybe

**Comment by csukuangfj** [comment, marked as answer]
Could you explain "not at all effective"?

The current approach is to cancel previous runs within the same PR.

**Reply by manickavela29** [reply]
Understood!

It is already in its best config with free tier.
My bad!
Thank you
